src/vs/platform/theme/common/colorRegistry.ts

```diff
@@ -314,14 +314,18 @@ export const editorBackground = registerColor('editor.background', { light: '#ff
  */
 export const editorForeground = registerColor('editor.foreground', { light: '#333333', dark: '#BBBBBB', hcDark: Color.white, hcLight: foreground }, nls.localize('editorForeground', "Editor default foreground color."));
 
+/**
+ * Sticky scroll
+ */
+export const stickyScrollBackground = registerColor('stickyScroll.background', { light: darken(editorBackground, 0.2), dark: lighten(editorBackground, 0.2), hcDark: lighten(editorBackground, 0.2), hcLight: darken(editorBackground, 0.2) }, nls.localize('stickyScrollBackground', "Sticky scroll background color"));
```

Comment:
IMHO we should find a better name for these colors, to at least contain the prefix `editor`, otherwise users might not realise this is a color that only applies to the editor.